internal-registry/che-incubator/command-line-terminal-dev/4.5.0/meta.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+publisher: che-incubator
+name: command-line-terminal
```

Comment:
```suggestion
name: dev-command-line-terminal
```

Reply:
Updated to `command-line-terminal-dev` to match path.